configs/default/SJTE-AOCODARCF722.config

```diff
@@ -0,0 +1,137 @@
+# Betaflight / AOCODARCF722 (AOF7) 4.3.0 Mar  3 2022 / 17:36:26 (ccd9508fc) MSP API: 1.44
+
+board_name AOCODARCF722
+manufacturer_id SJTE
+
+# resources
+resource BEEPER 1 C13
+resource MOTOR 1 C06
+resource MOTOR 2 C07
+resource MOTOR 3 C08
+resource MOTOR 4 C09
+resource MOTOR 5 B00
+resource MOTOR 6 B01
+resource MOTOR 7 B06
+resource MOTOR 8 B07
+resource PPM 1 A10
+resource LED_STRIP 1 A00
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 C10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 5 C12
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 C11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 5 D02
+resource I2C_SCL 1 B08
+resource I2C_SCL 2 B10
+resource I2C_SDA 1 B09
+resource I2C_SDA 2 B11
+resource LED 1 C14
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 B03
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 B04
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource ADC_BATT 1 C02
+resource ADC_RSSI 1 C03
+resource ADC_CURR 1 C01
+resource FLASH_CS 1 C00
+resource OSD_CS 1 A04
+resource GYRO_EXTI 1 C04
+resource GYRO_EXTI 2 A08
+resource GYRO_CS 1 B12
+resource GYRO_CS 2 A13
+resource USB_DETECT 1 C15
+
+# timer
+timer A10 AF1
+# pin A10: TIM1 CH3 (AF1)
+timer C06 AF3
+# pin C06: TIM8 CH1 (AF3)
+timer C07 AF3
+# pin C07: TIM8 CH2 (AF3)
+timer C08 AF3
+# pin C08: TIM8 CH3 (AF3)
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer A00 AF1
+# pin A00: TIM2 CH1 (AF1)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin A10 0
+# pin A10: DMA2 Stream 6 Channel 0
+dma pin C06 0
+# pin C06: DMA2 Stream 2 Channel 0
+dma pin C07 0
+# pin C07: DMA2 Stream 2 Channel 0
+dma pin C08 1
+# pin C08: DMA2 Stream 4 Channel 7
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin A00 0
+# pin A00: DMA1 Stream 5 Channel 3
+
+# feature
+feature OSD
+
+# serial
+serial 20 1 115200 57600 0 115200
+
+# master
+set gyro_to_use = BOTH
```

Comment:
Doesnt work with 2 diferent gyros.

Reply:
i'm very sorry! Step CLI "diff all bare" is missing.
I have changed it,please check again.
thank you.